src/skins/vector/css/matrix-react-sdk/views/messages/MTextBody.css

```diff
@@ -13,12 +13,3 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 */
-
-.mx_MTextBody {
-    white-space: pre-wrap;
-}
-
-.mx_MTextBody pre{
```

Comment:
Why? Limiting the size of pre blocks has been very popular since its introduction, so that when folks paste a huge wodge of code into the room, it automatically gets put inside a scrollblock section.